lib/views/admin_portal/admin_portal.dart

```diff
@@ -11,6 +12,10 @@ class AdminPortal extends StatelessWidget {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(actions: [
+        IconButton(
+          onPressed: () => navigator.push(const ProfilesScreen()),
+          icon: const Icon(Icons.person),
+        ),
```

Comment:
Currently, the admin portal has a "profile" and "settings" button, and the participant home screen has a "logout" button.

My guess is that "settings" and "logout" will eventually be moved into the profile screen, but that's something we can decide as the app becomes more fleshed out.